am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/RevokeAccessIntegrationTest.java

```diff
@@ -118,6 +119,64 @@ void whenRevokingResourceAccessThatDoesNotExistNoErrorExpected() {
         assertThat(databaseHelper.countExplicitPermissions(resourceId)).isEqualTo(0);
     }
 
+    @Test
+    void whenRevokingResourceWithNonExistentRelationshipShouldNotRemoveAnyRecordFromDatabase() {
+        grantExplicitResourceAccess(resourceId, "");
```

Comment:
Why not to use `ROOT_ATTRIBUTE` constant here? If not feasible at least be consistent.

Reply:
👍